lib/pf/api.pm

```diff
@@ -912,8 +913,12 @@ sub dynamic_register_node : Public {
 
 sub fingerbank_process : Public {
     my ( $class, $args ) = @_;
-
-    return (pf::fingerbank::process($args));
+    my $filter = pf::access_filter::dhcp->new;
+    my $rule = $filter->filter('DhcpFingerbank', $args);
+    if (!$rule) {
+        delete $args->{'computer_name'};
+        return (pf::fingerbank::process($args));
+    }
```

Comment:
I think we should set a return if it does not match.
We may not check for it now but who knows what we will do with it someday.
We could just return undef for now.